src/components/Features/Features.css

```diff
@@ -0,0 +1,92 @@
+.features {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 62px 50px;
+  gap: 10px;
+}
+
+.featureHead {
+  color: #ffffff;
+}
+
+.head {
+  text-align: center;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 52px;
+  line-height: 62px;
+  letter-spacing: 0.2px;
+}
+
+.featuresHeader {
+  width: 100%;
+  padding: 10px;
+}
+
+.featureSubHead .featureText {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 30px;
+  text-align: center;
+  letter-spacing: 0.2px;
+}
+
+.featureSubHead {
+  text-align: center;
+  padding: 15px;
+}
+.featuresFrame {
+  display: flex;
+  padding: 50px 130px;
+}
+.frameContent {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 40px 20px;
```

Comment:
I think you can use `width` here of maybe 50%, 
and a `justify-content: flex-start`. then a `padding-top` for adjustments